docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md

```diff
@@ -0,0 +1,108 @@
+---
+aliases:
+  - /docs/grafana/latest/setup-grafana/configure-grafana/feature-toggles/
+description: Learn about toggles for experimental and beta features, which you can enable or disable.
+title: Configure feature toggles
+weight: 150
+---
+
+# Configure feature toggles
+
+Feature toggles, also known as feature flags, are used for experimental or beta features in Grafana. Although we do not recommend that you use these features in production, you can turn on feature toggles to try out new functionality in development or test environments.
+
+This page contains a list of available feature toggles. To learn how to turn on feature toggles, refer to our [Configure Grafana documentation]({{< relref "../_index.md/#feature_toggles" >}}). Feature toogles are also available to Grafana Cloud Advanced customers - if you use Grafana Cloud Advanced, you can open a support ticket specifying the feature toggles and stack you would like them enabled
```

Comment:
I would add the instructions for turning on a feature toggle here. It duplicates content, but a very small amount. It'll provide a nicer user experience not to navigate to another set of docs.